js/add-books.js

```diff
@@ -1,51 +1,94 @@
-const title = document.querySelector('#title');
-const author = document.querySelector('#author');
-const add = document.querySelector('.submit');
-const allBooks = document.querySelector('#all-books');
-
-const bookStore = JSON.parse(localStorage.getItem('books')) || [];
-
-function showBooks() {
-  const displayBooks = bookStore.map((book, index) => `
-    <article class="books">
-      <p>${book.title}</p>
-      <p>${book.author}</p>
-      <button class="remove" onclick="removeBook(${index})">Remove</button>
-      <hr />
-    </article>
-`);
-  allBooks.innerHTML = displayBooks.join('');
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please avoid turning off linters they are used to make our code work better and make it easier to read.

Reply:
@mikethreels Hello,

I think that everything works perfectly! But in the project requirement, they said we could use some rules to avoid some errors in the linters...

<img width="1254" alt="Screen Shot 2023-07-04 at 5 12 28 PM" src="https://github.com/jecode93/awesome-book/assets/45808742/8b112e03-04f6-484e-adb8-c139e7e81163">